sql/vms.sql

```diff
@@ -67,6 +67,28 @@ CREATE TABLE `dbEventVolunteers` (
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `dbTags`
+--
+
+CREATE TABLE `dbTags` (
```

Comment:
Best to change to CREATE TABLE IF NOT EXISTS so that MySQL can import it without causing errors if there's a pre-existing table with that name.
Then, any edits to the table could be put in ALTER TABLE